docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+# Replication manager for Percona XtraDB Cluster
+
+The replication manager script helps manage source-source replication between multiple Percona XtraDB Cluster clusters.
```

Comment:
What is source-source replication? perhaps it makes sense to explain it either here, or add a new entry to glossary and reference to it from this doc

Reply:
Changed to multi-source and added to Glossary.